### ISSUE TITLE
[FR] Quote new line separation in editing vs reading modes

### ISSUE THREAD

**Issue by mbentley** [damiankorcz/Prism-Theme#118]
**Is your feature request related to a problem? Please describe.**
I am not sure if this is a design decision or not; when in reading mode vs editing mode, multi-line quotes are separated by a border in editing mode but I would expect them to display like they do in reading mode with no separation of the quote.

Raw markdown:
```
> This is a quote.  This could be a much longer sentence if I really wanted it to be so here is some more info that I could possible put in this.  I hope this is long enough because I am tired of typing.
> New line of a quote

> New quote, separated by blank line
```

Editing mode:
![screenshot 2023-01-19 at 10 41 20 AM](https://user-images.githubusercontent.com/414445/213487043-d0250703-d548-45b7-ba28-0349ae317816.png)

Reading mode:
![screenshot 2023-01-19 at 10 41 24 AM](https://user-images.githubusercontent.com/414445/213487076-867d7138-cc78-4c1f-a0b2-5baca5e8e96d.png)

**Describe the solution you'd like**
Editing mode would look like reading mode.

**Describe alternatives you've considered**
I was looking at trying to change this using a CSS snippet but I was having issues trying to do so.

**Additional context**
Screen shots above.

**Comment by damiankorcz** [comment]
Hi @mbentley,
I'm aware of this and will likely look for an alternative design that I can keep consistent across modes. At the moment there is no way for me to target the start and end of a blockquote individually in editing modes. So the start, in-between and the end all have to share the same code. which results in the unwanted top and bottom borders when using several lines. Will likely resort to possibly adding a subtle background colour and only doing the side borders. Will have to think about this one 😅

---

P.S. I've had a similar issue with Inline code but I've managed to get some help with resolving that and it will be in the next update. In that case there was a way to differentiate the parts that make it up so the bordered design can stay.

How that looks currently:
![image](https://user-images.githubusercontent.com/7290097/213490354-0755c82e-3975-4687-b390-2bd85e922856.png)


Solution 1:
![image](https://user-images.githubusercontent.com/7290097/213490098-35bf446e-217f-4f60-b5d7-7bbdfd22c73c.png)

Solution 2:
![image](https://user-images.githubusercontent.com/7290097/213490167-457f26e8-4846-4703-b759-d432d3ae3baa.png)

**Comment by damiankorcz** [comment]
Hi @mbentley,
This is now addressed in Prism 3.2.1. I've basically reverted the design to what the default theme uses with an addition of a different background colour to help differentiate it from other content. I've had too many difficulties doing anything else while making it consistent across all editor modes.
